content/images/orcid.svg

```diff
@@ -1,4 +1,4 @@
 <!-- Modified from: https://figshare.com/articles/ORCID_iD_icon_graphics/5008697 -->

```

Comment:
This also needs to be updated to comment from https://github.com/manubot/resources/pull/1